app/controllers/dradis/plugins/csv/upload_controller.rb

```diff
@@ -2,24 +2,51 @@ module Dradis::Plugins::CSV
   class UploadController < ::AuthenticatedController
     include ProjectScoped
 
-    before_action :load_attachment, only: [:new]
+    before_action :load_attachment, only: [:new, :create]
 
     def new
       @default_columns = ['Unique Identifier', 'Column Header From File', 'Type', 'Field in Dradis']
 
       @headers = ::CSV.open(@attachment.fullpath, &:readline)
+
+      @log_uid = Log.new.uid + 1
```

Comment:
Why are we adding a `+ 1` here when the Log initialization already does that for us?

Reply:
Gotcha, will remove it. I thought it didn't because there's a `+ 1` in `uploads.js.coffee` https://github.com/secroots/dradis-pro/blob/develop/app/assets/javascripts/tylium/modules/uploads.js.coffee#L28 :man_shrugging: